src/Core/ClassInfo.php

```diff
@@ -87,16 +81,26 @@ public static function exists($class)
      */
     public static function hasTable($tableName)
```

Comment:
Probably worth adding a PHPDoc description for this method and explicitly stating it's cached.
Something like:

> Get the list of tables in the database. Unlike DBSchemaManager::tableList(), this is cached.